pkgs/development/compilers/sdcc/zsdcc.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, sdcc, fetchFromGitHub, ... }:
+# https://www.z88dk.org/wiki/doku.php?id=temp:front#sdcc1
+# https://github.com/z88dk/z88dk/blob/master/src/zsdcc/readme.md
+
+sdcc.overrideAttrs (oldAttrs: rec {
+  version = "2018-02-24";
+  name = "zsdcc-unstable-${version}"; # ${oldAttrs.version}
```

Comment:
I'm not happy about packaging unstable versions. To me, that means it's not ready for Nixpkgs / end users.

Reply:
there is no release of zsdcc, and it's not target end users, but users of z88dk that wants to work with new clibs.

Reply:
there is no release of zsdcc, and it's not target end users, but users of z88dk that wants to work with new clibs.